tools/replace-next-version-tag.sh

```diff
@@ -85,7 +85,7 @@ for FILE in $(git ls-files "projects/$SLUG/"); do
 
 	sed -i.bak -E -e 's!(@since|@deprecated( +[sS]ince)?)( +)\$\$next-version\$\$!\1\3'"$VE"'!g' "$FILE"
 	rm "$FILE.bak" # We need a backup file because macOS requires it.
-	sed -i.bak -E -e $'s!(^\t*_deprecated_(function|constructor|file|argument|hook|class)\\( .*, \'[^\']*)\\$\\$next-version\\$\\$\'!\\1'"$VE"$'\'!g' "$FILE"
+	sed -i.bak -E -e $'s!(^\t*(_deprecated_(function|constructor|file|argument|hook|class)|_doing_it_wrong)\\( .*, \'[^\']*)\\$\\$next-version\\$\\$\'!\\1'"$VE"$'\'!g' "$FILE"
```

Comment:
Discussion about this change - p1740055882433239-slack-C05Q5HSS013

Reply:
@Automattic/jetpack-garage you might want to check this. I hope that previous back reference needs to be checked.

Reply:
There's at least an extra `)` in the regex now.

Reply:
Looks fine to me. It might be worthwhile to send this commit as a separate PR.

Reply:
I reverted that change as it's not applicable after implementing the suggestion in https://github.com/Automattic/jetpack/pull/41911#discussion_r1963575742